scan/src/rpc/TxHook.re

```diff
@@ -42,6 +42,14 @@ module Coin = {
   let newCoin = (denom, amount) => {denom, amount};
 
   let getDescription = coin => (coin.amount |> Format.fPretty) ++ " " ++ coin.denom;
+
+  let getAmount = coins => {
+    let coinOpt = coins->Belt_List.get(0);
```

Comment:
Assume that it has only one element?

Reply:
Its name is misleading. It should be named `getFeeAmount`. Currently, I think we don't need to support multiple fees.